test/parse_strategy_test.rb

```diff
@@ -17,11 +17,12 @@ class ParseStrategySyncTest < BaseTest
 
       representer!(:module => mod, :name => :song_representer) do
         property :title
-        self.representation_wrap = :song if format == :xml
```

Comment:
This changes the test, can you please revert that? In this test I want to make sure `representation_wrap` works and not `:as`.